Jenkinsfile

```diff
@@ -0,0 +1,24 @@
+// For ci.jenkins.io
+// https://github.com/jenkins-infra/documentation/blob/master/ci.adoc
+
+def mavenName = 'mvn'
+def jdkName = 'jdk8'
+
+node('highram&&docker') {
+    withEnv([
+        "JAVA_HOME=${tool jdkName}",
+        "PATH+MAVEN=${tool mavenName}/bin:${env.JAVA_HOME}/bin"
+    ]) {
+        changelog scm
+
+        sh 'docker build --build-arg=uid=$(id -u) --build-arg=gid=$(id -g) -t jenkins/ath src/main/resources/ath-container'
+        docker.image('jenkins/ath').inside {
+            sh '''
+                eval $(./vnc.sh > /dev/null 2>&1)
```

Comment:
I think it should be:

`eval $(./vnc.sh 2>/dev/null)`

otherwise the `echo export BROWSER_DISPLAY=$display` has no effect